packages/expo-dev-menu/app/components/Main.tsx

```diff
@@ -210,7 +192,23 @@ export function Main({ registeredCallbacks = [], isDevice }: MainProps) {
             </View>
           )}
 
+          <View rounded="large" bg="default" margin="small">
+            <SettingsRowButton label="Reload" icon={<RefreshIcon />} onPress={actions.reload} />
+          </View>
+
           <View mx="small">
+            {isDevLauncherInstalled && (
+              <>
+                <View roundedTop="large" bg="default">
+                  <SettingsRowButton
+                    label="Go home"
+                    icon={<HomeFilledIcon />}
+                    onPress={actions.navigateToLauncher}
+                  />
+                </View>
+                <Divider />
+              </>
+            )}
             <View roundedTop="large" bg="default">
```

Comment:
Nit
<img width="1106" alt="image" src="https://github.com/expo/expo/assets/11707729/1cf2b662-4e91-461a-b140-0caa47023e7d">

```suggestion
            <View  bg="default">
```

Reply:
Oops! I can't believe I missed this. Updated ✔️